CRITICAL - 2024-08-02 02:36:27 --> TypeError: App\Controllers\Home::index(): Return value must be of type string, array returned
[Method: GET, Route: /]
in APPPATH/Controllers/Home.php on line 20.
 1 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->index()
 2 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 03:29:45 --> ErrorException: Undefined property: App\Controllers\Payments::$payments
[Method: POST, Route: payments/getData]
in APPPATH/Controllers/Payments.php on line 56.
 1 APPPATH/Controllers/Payments.php(56): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Payments::$payments', '/opt/lampp/htdocs/task_vm/app/Controllers/Payments.php', 56)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->getData()
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 03:31:24 --> ErrorException: Undefined property: App\Controllers\Payments::$payments
[Method: POST, Route: payments/getData]
in APPPATH/Controllers/Payments.php on line 56.
 1 APPPATH/Controllers/Payments.php(56): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Payments::$payments', '/opt/lampp/htdocs/task_vm/app/Controllers/Payments.php', 56)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->getData()
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 03:35:56 --> ErrorException: Undefined property: App\Controllers\Home::$payments
[Method: GET, Route: /]
in APPPATH/Controllers/Home.php on line 22.
 1 APPPATH/Controllers/Home.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Home::$payments', '/opt/lampp/htdocs/task_vm/app/Controllers/Home.php', 22)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->index()
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 03:36:09 --> ErrorException: Undefined variable $payments
[Method: GET, Route: /]
in APPPATH/Controllers/Home.php on line 22.
 1 APPPATH/Controllers/Home.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $payments', '/opt/lampp/htdocs/task_vm/app/Controllers/Home.php', 22)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->index()
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 03:40:00 --> mysqli_sql_exception: Column 'name' in field list is ambiguous in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `payment...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `payment...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `payment...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `payment...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Home.php(27): CodeIgniter\Database\BaseBuilder->get()
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Home->index()
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 03:40:00 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'name' in field list is ambiguous
[Method: GET, Route: /]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `payments`.*, `name` as `customer_name`, `model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id`', [], false)
 2 APPPATH/Controllers/Home.php(27): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->index()
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 03:40:00 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'name' in field list is ambiguous
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `payments`.*, `name` as `customer_name`, `model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id`')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `payments`.*, `name` as `customer_name`, `model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id`')
 3 SYSTEMPATH/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `payments`.*, `name` as `customer_name`, `model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id`', [], false)
 4 APPPATH/Controllers/Home.php(27): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 03:40:00 --> [Caused by] mysqli_sql_exception: Column 'name' in field list is ambiguous
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `payments`.*, `name` as `customer_name`, `model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id`', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `payments`.*, `name` as `customer_name`, `model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id`')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `payments`.*, `name` as `customer_name`, `model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id`')
 4 SYSTEMPATH/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `payments`.*, `name` as `customer_name`, `model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id`', [], false)
 5 APPPATH/Controllers/Home.php(27): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 03:44:18 --> mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'as `vehicle_name`' at line 4 in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `payment...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `payment...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `payment...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `payment...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Home.php(27): CodeIgniter\Database\BaseBuilder->get()
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Home->index()
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 03:44:18 --> CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'as `vehicle_name`' at line 4
[Method: GET, Route: /]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `payments`.*, `customers`.`name` as `customer_name`, `vehicles`.`model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id ,vehicles`.`name` as `vehicle_name`', [], false)
 2 APPPATH/Controllers/Home.php(27): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->index()
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 03:44:18 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'as `vehicle_name`' at line 4
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `payments`.*, `customers`.`name` as `customer_name`, `vehicles`.`model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id ,vehicles`.`name` as `vehicle_name`')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `payments`.*, `customers`.`name` as `customer_name`, `vehicles`.`model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id ,vehicles`.`name` as `vehicle_name`')
 3 SYSTEMPATH/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `payments`.*, `customers`.`name` as `customer_name`, `vehicles`.`model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id ,vehicles`.`name` as `vehicle_name`', [], false)
 4 APPPATH/Controllers/Home.php(27): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->index()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 03:44:18 --> [Caused by] mysqli_sql_exception: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'as `vehicle_name`' at line 4
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('SELECT `payments`.*, `customers`.`name` as `customer_name`, `vehicles`.`model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id ,vehicles`.`name` as `vehicle_name`', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `payments`.*, `customers`.`name` as `customer_name`, `vehicles`.`model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id ,vehicles`.`name` as `vehicle_name`')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `payments`.*, `customers`.`name` as `customer_name`, `vehicles`.`model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id ,vehicles`.`name` as `vehicle_name`')
 4 SYSTEMPATH/Database/BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT `payments`.*, `customers`.`name` as `customer_name`, `vehicles`.`model` as `vehicle_model`
FROM `payments`
LEFT JOIN `customers` ON `payments`.`customer_id` = `customers`.`id`
LEFT JOIN `vehicles` ON `payments`.`vehicle_id` = `vehicles`.`id ,vehicles`.`name` as `vehicle_name`', [], false)
 5 APPPATH/Controllers/Home.php(27): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 03:50:29 --> ErrorException: Attempt to read property "customer_name" on array
[Method: POST, Route: payments/getData]
in APPPATH/Controllers/Payments.php on line 70.
 1 APPPATH/Controllers/Payments.php(70): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "customer_name" on array', '/opt/lampp/htdocs/task_vm/app/Controllers/Payments.php', 70)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->getData()
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 03:51:54 --> ErrorException: Attempt to read property "customer_name" on array
[Method: POST, Route: payments/getData]
in APPPATH/Controllers/Payments.php on line 70.
 1 APPPATH/Controllers/Payments.php(70): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "customer_name" on array', '/opt/lampp/htdocs/task_vm/app/Controllers/Payments.php', 70)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->getData()
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 04:04:04 --> mysqli_sql_exception: Column 'name' cannot be null in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `cu...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cu...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cu...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cu...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Customers->addData()
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 04:04:04 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'name' cannot be null
[Method: POST, Route: customers/addData]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (:name:, :email:, :mobile_number:)', [...], false)
 2 APPPATH/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->addData()
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 04:04:04 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'name' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (NULL, NULL, NULL)')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (NULL, NULL, NULL)')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (:name:, :email:, :mobile_number:)', [...], false)
 4 APPPATH/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->addData()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 04:04:04 --> [Caused by] mysqli_sql_exception: Column 'name' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (NULL, NULL, NULL)', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (NULL, NULL, NULL)')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (NULL, NULL, NULL)')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (:name:, :email:, :mobile_number:)', [...], false)
 5 APPPATH/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->addData()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 04:10:03 --> mysqli_sql_exception: Column 'name' cannot be null in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `cu...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cu...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cu...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cu...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Customers->addData()
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 04:10:03 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'name' cannot be null
[Method: POST, Route: customers/addData]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (:name:, :email:, :mobile_number:)', [...], false)
 2 APPPATH/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->addData()
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 04:10:03 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'name' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (NULL, NULL, NULL)')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (NULL, NULL, NULL)')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (:name:, :email:, :mobile_number:)', [...], false)
 4 APPPATH/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->addData()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 04:10:03 --> [Caused by] mysqli_sql_exception: Column 'name' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (NULL, NULL, NULL)', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (NULL, NULL, NULL)')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (NULL, NULL, NULL)')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (:name:, :email:, :mobile_number:)', [...], false)
 5 APPPATH/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->addData()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 04:10:58 --> mysqli_sql_exception: Column 'email' cannot be null in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `cu...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cu...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cu...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cu...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Customers->addData()
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 04:10:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'email' cannot be null
[Method: POST, Route: customers/addData]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (:name:, :email:, :mobile_number:)', [...], false)
 2 APPPATH/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->addData()
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 04:10:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'email' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (\'son\', NULL, NULL)')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (\'son\', NULL, NULL)')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (:name:, :email:, :mobile_number:)', [...], false)
 4 APPPATH/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->addData()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 04:10:58 --> [Caused by] mysqli_sql_exception: Column 'email' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (\'son\', NULL, NULL)', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (\'son\', NULL, NULL)')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (\'son\', NULL, NULL)')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (:name:, :email:, :mobile_number:)', [...], false)
 5 APPPATH/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->addData()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 04:15:34 --> mysqli_sql_exception: Column 'email' cannot be null in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `cu...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cu...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cu...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cu...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Customers->addData()
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 04:15:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'email' cannot be null
[Method: POST, Route: customers/addData]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (:name:, :email:, :mobile_number:)', [...], false)
 2 APPPATH/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->addData()
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 04:15:34 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'email' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (\'son\', NULL, NULL)')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (\'son\', NULL, NULL)')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (:name:, :email:, :mobile_number:)', [...], false)
 4 APPPATH/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->addData()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 04:15:34 --> [Caused by] mysqli_sql_exception: Column 'email' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (\'son\', NULL, NULL)', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (\'son\', NULL, NULL)')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (\'son\', NULL, NULL)')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `customers` (`name`, `email`, `mobile_number`) VALUES (:name:, :email:, :mobile_number:)', [...], false)
 5 APPPATH/Controllers/Customers.php(36): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->addData()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 04:16:47 --> mysqli_sql_exception: Unknown column 'name' in 'field list' in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `pa...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pa...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pa...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pa...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Payments.php(39): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Payments->addData()
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 04:16:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'name' in 'field list'
[Method: POST, Route: payments/addData]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`, `name`) VALUES (:customer_id:, :vehicle_id:, :amount:, :name:)', [...], false)
 2 APPPATH/Controllers/Payments.php(39): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->addData()
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 04:16:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'name' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`, `name`) VALUES (\'1\', \'3\', \'60000\', NULL)')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`, `name`) VALUES (\'1\', \'3\', \'60000\', NULL)')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`, `name`) VALUES (:customer_id:, :vehicle_id:, :amount:, :name:)', [...], false)
 4 APPPATH/Controllers/Payments.php(39): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->addData()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 04:16:47 --> [Caused by] mysqli_sql_exception: Unknown column 'name' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`, `name`) VALUES (\'1\', \'3\', \'60000\', NULL)', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`, `name`) VALUES (\'1\', \'3\', \'60000\', NULL)')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`, `name`) VALUES (\'1\', \'3\', \'60000\', NULL)')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`, `name`) VALUES (:customer_id:, :vehicle_id:, :amount:, :name:)', [...], false)
 5 APPPATH/Controllers/Payments.php(39): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->addData()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 06:12:15 --> mysqli_sql_exception: Column 'customer_id' cannot be null in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `pa...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pa...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pa...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pa...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Payments.php(39): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Payments->addData()
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 06:12:16 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'customer_id' cannot be null
[Method: POST, Route: payments/addData]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (:customer_id:, :vehicle_id:, :amount:)', [...], false)
 2 APPPATH/Controllers/Payments.php(39): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->addData()
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 06:12:16 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'customer_id' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (NULL, NULL, \'3\')')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (NULL, NULL, \'3\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (:customer_id:, :vehicle_id:, :amount:)', [...], false)
 4 APPPATH/Controllers/Payments.php(39): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->addData()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 06:12:16 --> [Caused by] mysqli_sql_exception: Column 'customer_id' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (NULL, NULL, \'3\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (NULL, NULL, \'3\')')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (NULL, NULL, \'3\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (:customer_id:, :vehicle_id:, :amount:)', [...], false)
 5 APPPATH/Controllers/Payments.php(39): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->addData()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 06:18:47 --> mysqli_sql_exception: Column 'customer_id' cannot be null in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `pa...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pa...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pa...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pa...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Payments.php(39): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Payments->addData()
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 06:18:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'customer_id' cannot be null
[Method: POST, Route: payments/addData]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (:customer_id:, :vehicle_id:, :amount:)', [...], false)
 2 APPPATH/Controllers/Payments.php(39): CodeIgniter\Database\BaseBuilder->insert([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->addData()
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 06:18:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'customer_id' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (NULL, NULL, \'300\')')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (NULL, NULL, \'300\')')
 3 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (:customer_id:, :vehicle_id:, :amount:)', [...], false)
 4 APPPATH/Controllers/Payments.php(39): CodeIgniter\Database\BaseBuilder->insert([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->addData()
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 06:18:47 --> [Caused by] mysqli_sql_exception: Column 'customer_id' cannot be null
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (NULL, NULL, \'300\')', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (NULL, NULL, \'300\')')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (NULL, NULL, \'300\')')
 4 SYSTEMPATH/Database/BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `payments` (`customer_id`, `vehicle_id`, `amount`) VALUES (:customer_id:, :vehicle_id:, :amount:)', [...], false)
 5 APPPATH/Controllers/Payments.php(39): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->addData()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 06:29:41 --> ErrorException: Undefined property: App\Controllers\Payments::$datatable
[Method: POST, Route: payments/delete/8]
in APPPATH/Controllers/Payments.php on line 104.
 1 APPPATH/Controllers/Payments.php(104): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Payments::$datatable', '/opt/lampp/htdocs/task_vm/app/Controllers/Payments.php', 104)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->delete('8')
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 06:29:44 --> ErrorException: Undefined property: App\Controllers\Payments::$datatable
[Method: POST, Route: payments/delete/8]
in APPPATH/Controllers/Payments.php on line 104.
 1 APPPATH/Controllers/Payments.php(104): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Payments::$datatable', '/opt/lampp/htdocs/task_vm/app/Controllers/Payments.php', 104)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->delete('8')
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 06:31:12 --> ErrorException: Undefined property: App\Controllers\Payments::$datatable
[Method: POST, Route: payments/delete/8]
in APPPATH/Controllers/Payments.php on line 104.
 1 APPPATH/Controllers/Payments.php(104): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Payments::$datatable', '/opt/lampp/htdocs/task_vm/app/Controllers/Payments.php', 104)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->delete('8')
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 06:31:14 --> ErrorException: Undefined property: App\Controllers\Payments::$datatable
[Method: POST, Route: payments/delete/8]
in APPPATH/Controllers/Payments.php on line 104.
 1 APPPATH/Controllers/Payments.php(104): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Payments::$datatable', '/opt/lampp/htdocs/task_vm/app/Controllers/Payments.php', 104)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->delete('8')
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 06:31:31 --> ErrorException: Undefined property: App\Controllers\Payments::$datatable
[Method: POST, Route: payments/delete/7]
in APPPATH/Controllers/Payments.php on line 104.
 1 APPPATH/Controllers/Payments.php(104): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\Payments::$datatable', '/opt/lampp/htdocs/task_vm/app/Controllers/Payments.php', 104)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->delete('7')
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 07:07:45 --> ErrorException: Undefined property: stdClass::$name
[Method: GET, Route: payments/edit/1]
in APPPATH/Views/template/customers/editpage.php on line 7.
 1 APPPATH/Views/template/customers/editpage.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$name', '/opt/lampp/htdocs/task_vm/app/Views/template/customers/editpage.php', 7)
 2 SYSTEMPATH/View/View.php(234): include('/opt/lampp/htdocs/task_vm/app/Views/template/customers/editpage.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1169): CodeIgniter\View\View->render('template/customers/editpage', [], true)
 5 APPPATH/Controllers/Payments.php(94): view('template/customers/editpage', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->edit('1')
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 07:10:14 --> ErrorException: Undefined property: stdClass::$name
[Method: GET, Route: payments/edit/2]
in APPPATH/Views/template/customers/editpage.php on line 7.
 1 APPPATH/Views/template/customers/editpage.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$name', '/opt/lampp/htdocs/task_vm/app/Views/template/customers/editpage.php', 7)
 2 SYSTEMPATH/View/View.php(234): include('/opt/lampp/htdocs/task_vm/app/Views/template/customers/editpage.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1169): CodeIgniter\View\View->render('template/customers/editpage', [], true)
 5 APPPATH/Controllers/Payments.php(94): view('template/customers/editpage', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->edit('2')
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 07:12:45 --> ErrorException: Undefined property: stdClass::$name
[Method: GET, Route: payments/edit/2]
in APPPATH/Views/template/customers/editpage.php on line 7.
 1 APPPATH/Views/template/customers/editpage.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$name', '/opt/lampp/htdocs/task_vm/app/Views/template/customers/editpage.php', 7)
 2 SYSTEMPATH/View/View.php(234): include('/opt/lampp/htdocs/task_vm/app/Views/template/customers/editpage.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1169): CodeIgniter\View\View->render('template/customers/editpage', [], true)
 5 APPPATH/Controllers/Payments.php(94): view('template/customers/editpage', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->edit('2')
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 07:13:43 --> ErrorException: Undefined property: stdClass::$mobile_number
[Method: GET, Route: payments/edit/1]
in APPPATH/Views/template/payments/editpage.php on line 36.
 1 APPPATH/Views/template/payments/editpage.php(36): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$mobile_number', '/opt/lampp/htdocs/task_vm/app/Views/template/payments/editpage.php', 36)
 2 SYSTEMPATH/View/View.php(234): include('/opt/lampp/htdocs/task_vm/app/Views/template/payments/editpage.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1169): CodeIgniter\View\View->render('template/payments/editpage', [], true)
 5 APPPATH/Controllers/Payments.php(94): view('template/payments/editpage', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->edit('1')
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 07:36:58 --> mysqli_sql_exception: Unknown column 'customer_id' in 'field list' in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `custome...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `custome...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `custome...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `custome...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Customers->update('9')
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 07:36:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'customer_id' in 'field list'
[Method: POST, Route: customers/update/9]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('9')
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 07:36:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'customer_id' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'300\'
WHERE `id` = \'9\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'300\'
WHERE `id` = \'9\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('9')
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 07:36:58 --> [Caused by] mysqli_sql_exception: Unknown column 'customer_id' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'300\'
WHERE `id` = \'9\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'300\'
WHERE `id` = \'9\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'300\'
WHERE `id` = \'9\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('9')
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 07:49:39 --> mysqli_sql_exception: Unknown column 'customer_id' in 'field list' in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `custome...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `custome...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `custome...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `custome...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Customers->update('9')
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 07:49:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'customer_id' in 'field list'
[Method: POST, Route: customers/update/9]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('9')
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 07:49:39 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'customer_id' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'300\'
WHERE `id` = \'9\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'300\'
WHERE `id` = \'9\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('9')
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 07:49:39 --> [Caused by] mysqli_sql_exception: Unknown column 'customer_id' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'300\'
WHERE `id` = \'9\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'300\'
WHERE `id` = \'9\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'300\'
WHERE `id` = \'9\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('9')
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 09:49:05 --> mysqli_sql_exception: Unknown column 'customer_id' in 'field list' in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `custome...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `custome...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `custome...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `custome...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Customers->update('9')
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 09:49:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'customer_id' in 'field list'
[Method: POST, Route: customers/update/9]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('9')
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 09:49:05 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'customer_id' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'3000000\'
WHERE `id` = \'9\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'3000000\'
WHERE `id` = \'9\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('9')
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 09:49:05 --> [Caused by] mysqli_sql_exception: Unknown column 'customer_id' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'3000000\'
WHERE `id` = \'9\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'3000000\'
WHERE `id` = \'9\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'3000000\'
WHERE `id` = \'9\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('9')
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 09:49:07 --> mysqli_sql_exception: Unknown column 'customer_id' in 'field list' in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `custome...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `custome...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `custome...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `custome...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Customers->update('9')
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 09:49:07 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'customer_id' in 'field list'
[Method: POST, Route: customers/update/9]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('9')
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 09:49:07 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'customer_id' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'2999999\'
WHERE `id` = \'9\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'2999999\'
WHERE `id` = \'9\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('9')
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 09:49:07 --> [Caused by] mysqli_sql_exception: Unknown column 'customer_id' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'2999999\'
WHERE `id` = \'9\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'2999999\'
WHERE `id` = \'9\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'2999999\'
WHERE `id` = \'9\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('9')
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 09:49:46 --> mysqli_sql_exception: Unknown column 'customer_id' in 'field list' in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `custome...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `custome...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `custome...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `custome...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Customers->update('2')
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 09:49:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'customer_id' in 'field list'
[Method: POST, Route: customers/update/2]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('2')
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 09:49:46 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'customer_id' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'3\', `mobile_number` = \'60003\'
WHERE `id` = \'2\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'3\', `mobile_number` = \'60003\'
WHERE `id` = \'2\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('2')
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 09:49:46 --> [Caused by] mysqli_sql_exception: Unknown column 'customer_id' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'3\', `mobile_number` = \'60003\'
WHERE `id` = \'2\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'3\', `mobile_number` = \'60003\'
WHERE `id` = \'2\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `customers` SET `customer_id` = \'1\', `vehicle_id` = \'3\', `mobile_number` = \'60003\'
WHERE `id` = \'2\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('2')
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 09:53:13 --> mysqli_sql_exception: Unknown column 'customer_id' in 'field list' in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `custome...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `custome...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `custome...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `custome...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Customers->update('2')
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 09:53:13 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'customer_id' in 'field list'
[Method: POST, Route: customers/update/2]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `id` = :id.1:, `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('2')
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 09:53:13 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'customer_id' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `customers` SET `id` = \'2\', `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'6000\'
WHERE `id` = \'2\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `customers` SET `id` = \'2\', `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'6000\'
WHERE `id` = \'2\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `id` = :id.1:, `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('2')
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 09:53:13 --> [Caused by] mysqli_sql_exception: Unknown column 'customer_id' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `customers` SET `id` = \'2\', `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'6000\'
WHERE `id` = \'2\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `customers` SET `id` = \'2\', `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'6000\'
WHERE `id` = \'2\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `customers` SET `id` = \'2\', `customer_id` = \'1\', `vehicle_id` = \'1\', `mobile_number` = \'6000\'
WHERE `id` = \'2\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `id` = :id.1:, `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('2')
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 09:55:55 --> mysqli_sql_exception: Unknown column 'customer_id' in 'field list' in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `custome...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `custome...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `custome...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `custome...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Customers->update('2')
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 09:55:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'customer_id' in 'field list'
[Method: POST, Route: customers/update/2]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('2')
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 09:55:55 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'customer_id' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'3\', `mobile_number` = \'6000\'
WHERE `id` = \'2\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'3\', `mobile_number` = \'6000\'
WHERE `id` = \'2\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('2')
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 09:55:55 --> [Caused by] mysqli_sql_exception: Unknown column 'customer_id' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'3\', `mobile_number` = \'6000\'
WHERE `id` = \'2\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'3\', `mobile_number` = \'6000\'
WHERE `id` = \'2\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `customers` SET `customer_id` = \'2\', `vehicle_id` = \'3\', `mobile_number` = \'6000\'
WHERE `id` = \'2\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `customers` SET `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Controllers/Customers.php(84): CodeIgniter\Database\BaseBuilder->update([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Customers->update('2')
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Customers))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-08-02 09:57:10 --> mysqli_sql_exception: Unknown column 'mobile_number' in 'field list' in /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php:312
Stack trace:
#0 /opt/lampp/htdocs/task_vm/system/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `payment...', 0)
#1 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `payment...')
#2 /opt/lampp/htdocs/task_vm/system/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `payment...')
#3 /opt/lampp/htdocs/task_vm/system/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `payment...', Array, false)
#4 /opt/lampp/htdocs/task_vm/app/Controllers/Payments.php(99): CodeIgniter\Database\BaseBuilder->update(Array)
#5 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(933): App\Controllers\Payments->update('9')
#6 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
#7 /opt/lampp/htdocs/task_vm/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /opt/lampp/htdocs/task_vm/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /opt/lampp/htdocs/task_vm/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /opt/lampp/htdocs/task_vm/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2024-08-02 09:57:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'mobile_number' in 'field list'
[Method: POST, Route: payments/update/9]
in SYSTEMPATH/Database/BaseConnection.php on line 688.
 1 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `payments` SET `id` = :id.1:, `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 2 APPPATH/Controllers/Payments.php(99): CodeIgniter\Database\BaseBuilder->update([...])
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->update('9')
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 09:57:10 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'mobile_number' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 317.
 1 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `payments` SET `id` = \'9\', `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'3007\'
WHERE `id` = \'9\'')
 2 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `payments` SET `id` = \'9\', `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'3007\'
WHERE `id` = \'9\'')
 3 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `payments` SET `id` = :id.1:, `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 4 APPPATH/Controllers/Payments.php(99): CodeIgniter\Database\BaseBuilder->update([...])
 5 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->update('9')
 6 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 7 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 09:57:10 --> [Caused by] mysqli_sql_exception: Unknown column 'mobile_number' in 'field list'
in SYSTEMPATH/Database/MySQLi/Connection.php on line 312.
 1 SYSTEMPATH/Database/MySQLi/Connection.php(312): mysqli->query('UPDATE `payments` SET `id` = \'9\', `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'3007\'
WHERE `id` = \'9\'', 0)
 2 SYSTEMPATH/Database/BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `payments` SET `id` = \'9\', `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'3007\'
WHERE `id` = \'9\'')
 3 SYSTEMPATH/Database/BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `payments` SET `id` = \'9\', `customer_id` = \'2\', `vehicle_id` = \'1\', `mobile_number` = \'3007\'
WHERE `id` = \'9\'')
 4 SYSTEMPATH/Database/BaseBuilder.php(2516): CodeIgniter\Database\BaseConnection->query('UPDATE `payments` SET `id` = :id.1:, `customer_id` = :customer_id:, `vehicle_id` = :vehicle_id:, `mobile_number` = :mobile_number:
WHERE `id` = :id:', [...], false)
 5 APPPATH/Controllers/Payments.php(99): CodeIgniter\Database\BaseBuilder->update([...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Payments->update('9')
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Payments))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 10:09:37 --> ErrorException: Attempt to read property "name" on array
[Method: GET, Route: /]
in APPPATH/Controllers/Home.php on line 31.
 1 APPPATH/Controllers/Home.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "name" on array', '/opt/lampp/htdocs/task_vm/app/Controllers/Home.php', 31)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->index()
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 10:13:27 --> ErrorException: Attempt to read property "amount" on array
[Method: GET, Route: /]
in APPPATH/Controllers/Home.php on line 32.
 1 APPPATH/Controllers/Home.php(32): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "amount" on array', '/opt/lampp/htdocs/task_vm/app/Controllers/Home.php', 32)
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->index()
 3 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 4 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 10:14:06 --> Error: Call to undefined method CodeIgniter\Database\MySQLi\Result::getResults()
[Method: GET, Route: /]
in APPPATH/Controllers/Home.php on line 27.
 1 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->index()
 2 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-08-02 10:32:57 --> ErrorException: Undefined variable $data
[Method: GET, Route: /]
in APPPATH/Views/index.php on line 42.
 1 APPPATH/Views/index.php(42): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $data', '/opt/lampp/htdocs/task_vm/app/Views/index.php', 42)
 2 SYSTEMPATH/View/View.php(234): include('/opt/lampp/htdocs/task_vm/app/Views/index.php')
 3 SYSTEMPATH/View/View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH/Common.php(1169): CodeIgniter\View\View->render('index', [], true)
 5 APPPATH/Controllers/Home.php(50): view('index', [...])
 6 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->index()
 7 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
WARNING - 2024-08-02 11:46:42 --> [DEPRECATED] Creation of dynamic property App\Controllers\Api::$dataCustomers is deprecated in APPPATH\Controllers\Api.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Api->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
WARNING - 2024-08-02 11:46:42 --> [DEPRECATED] Creation of dynamic property App\Controllers\Api::$dataCustomers is deprecated in APPPATH\Controllers\Api.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Api->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
WARNING - 2024-08-02 11:46:42 --> [DEPRECATED] Creation of dynamic property App\Controllers\Api::$dataVehicles is deprecated in APPPATH\Controllers\Api.php on line 16.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Api->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
WARNING - 2024-08-02 11:46:42 --> [DEPRECATED] Creation of dynamic property App\Controllers\Api::$dataVehicles is deprecated in APPPATH\Controllers\Api.php on line 16.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Api->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
WARNING - 2024-08-02 11:47:29 --> [DEPRECATED] Creation of dynamic property App\Controllers\Api::$dataCustomers is deprecated in APPPATH\Controllers\Api.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Api->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
WARNING - 2024-08-02 11:47:29 --> [DEPRECATED] Creation of dynamic property App\Controllers\Api::$dataVehicles is deprecated in APPPATH\Controllers\Api.php on line 16.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Api->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
WARNING - 2024-08-02 11:47:29 --> [DEPRECATED] Creation of dynamic property App\Controllers\Api::$dataCustomers is deprecated in APPPATH\Controllers\Api.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Api->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
WARNING - 2024-08-02 11:47:29 --> [DEPRECATED] Creation of dynamic property App\Controllers\Api::$dataVehicles is deprecated in APPPATH\Controllers\Api.php on line 16.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Api->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
WARNING - 2024-08-02 11:49:38 --> [DEPRECATED] Creation of dynamic property App\Controllers\Api::$dataCustomers is deprecated in APPPATH\Controllers\Api.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Api->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
WARNING - 2024-08-02 11:49:38 --> [DEPRECATED] Creation of dynamic property App\Controllers\Api::$dataVehicles is deprecated in APPPATH\Controllers\Api.php on line 16.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Api->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
WARNING - 2024-08-02 11:49:38 --> [DEPRECATED] Creation of dynamic property App\Controllers\Api::$dataCustomers is deprecated in APPPATH\Controllers\Api.php on line 15.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Api->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
WARNING - 2024-08-02 11:49:38 --> [DEPRECATED] Creation of dynamic property App\Controllers\Api::$dataVehicles is deprecated in APPPATH\Controllers\Api.php on line 16.
 1 SYSTEMPATH\CodeIgniter.php(903): App\Controllers\Api->__construct()
 2 SYSTEMPATH\CodeIgniter.php(500): CodeIgniter\CodeIgniter->createController()
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
